examples/allennlp/allennlp_simple.py

```diff
@@ -19,59 +19,66 @@
 """
 
 import os
+import random
 import shutil
 
 import allennlp
 import allennlp.data
 import allennlp.models
 import allennlp.modules
+import numpy
 import torch
 
 import optuna
+from optuna.integration import AllenNLPPruningCallback
 
 
 DEVICE = -1  # If you want to use GPU, use DEVICE = 0.
 MAX_DATA_SIZE = 3000
+MODEL_DIR = os.path.join(os.getcwd(), "result")
+TARGET_METRIC = "validation_accuracy"
```

Comment:
If you don't use this variable other than L.131, we don't need to define this IMO.

Reply:
It is also used in L128. (https://github.com/optuna/optuna/pull/1399/files#diff-68d489e90782fdeb5d3d9d0274162956R128)

3224ad7ec
And I revised the PR to use `TARGET_METRIC` in `trainer` as well.